core/tabs/applications-setup/dwmtitus-setup.sh

```diff
@@ -161,6 +161,13 @@ setupDisplayManager() {
     fi
 }
 
+
+batteryShown() {
+    if ls /sys/class/power_supply/ 2>/dev/null | grep -q '^BAT'; then
+        sed -i '35s/^\(.\{36\}\)/\1'" battery"'/' "$HOME/.local/dwm-titus/polybar/themes/minimal/config.ini"
+    fi
```

Comment:
```suggestion
    for p in /sys/class/power_supply/BAT*; do
        if [[ -e "$p" ]]; then
            sed -i '35s/^\(.\{36\}\)/\1 battery/' "$HOME/.local/dwm-titus/polybar/themes/minimal/config.ini"
        fi
    done
```
idk if that was ur intention but ur implementation would only run on the first file it matches. also [shellcheck](https://www.shellcheck.net/wiki/SC2010)